tembo-cli/tests/tomls/merge/second-instance.toml

```diff
@@ -0,0 +1,5 @@
+[instance]
```

Comment:
Can you please remove all but 1 settings in the overlay (this) file, I recommend "instance_name" as the only setting to have in this file. Also it might be more clear to name the file "overlay.toml" or "merge.toml"